src/app/page/component/dashboard-list/dashboard-list.component.html

```diff
@@ -2,9 +2,10 @@
   <div class="resource-header">
     <h2 class="list-header">{{ title }}</h2>
     <a
+      *ngIf="!routeSelected()"
       href="javascript:void(0)"
```

Comment:
A nice improvement for accessibility and Cmd+click support would be to use `routerLink` instead of a JavaScript href. That's a possible improvement for the resource cards as well.